charts/cert-manager-issuers/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
```

Comment:
I was literally looking for the missing `kind` for about minute before I figured out that this is not a Kubernetes object YAML. :D